### ISSUE TITLE
CUDA linking error

### ISSUE THREAD

**Issue by fabvio** [speedinghzl/pytorch-segmentation-toolbox#3]
Hi,
Thanks for sharing your code. I am trying to train using CUDA 9.2 and pytorch 0.4.1 on a NVIDIA Tesla v100. I modified the build.sh file including the architecture for the v100, and built the custom module. When I try to run the main training script, I receive this output:

`
ImportError: /home/fabio/pytorch-segmentation-toolbox/libs/_ext/__ext.so: undefined symbol: __cudaPopCallConfiguration
`

Do you have any idea on how to solve it?

**Comment by speedinghzl** [comment]
The problem may be due to the mismatch of CUDA version, maybe you can solve it by changing CUDA version to 9.0 or 8.0?

**Comment by fabvio** [comment]
Changing the CUDA version was not an option, unfortunately. I modified the model and used the traditional BN+ReLU layers included in pytorch. Should this get lower mIoU?

**Comment by speedinghzl** [comment]
Yes, traditional BN doesn't support synchronization mean and var across the GPUs which will degrade performance.
